### DISCUSSION TITLE
Feature: add a floating action button

### DISCUSSION THREAD

**Discussion by md-owes** [saadeghi/daisyui#3073] [Ideas / feature requests]
I am looking for a floating action button that resembles as shown [here](https://codepen.io/olton/pen/RRyWVp). This would cover many use cases in any product development, such as:

- Quick Access to Primary Actions (quickly adding a new task)
- Enhanced User Navigation (accessing recent searches, viewing saved places)
- Interactive Content Creation (creating new posts, uploading photos)
- Contextual Actions (composing new email, attaching files, accessing the contact list)
- Support and Assistance (customer support, FAQs, live chat)
- Dynamic Interaction in Dashboards (add new widgets, create reports, switch views)

and so many much more.

**Comment by saadeghi** [comment]
Related topic:  
https://github.com/saadeghi/daisyui/discussions/1677

Example:  
https://play.tailwindcss.com/RymHm7SohZ
